### ISSUE TITLE
ORM

### ISSUE THREAD

**Issue by hexaredecimal** [hexaredecimal/BlazingWebX#1]
Add an ORM API that follows the builder pattern and how the project works.

**Comment by hexaredecimal** [comment]
The ORM should be built on top of the JDBC package and follow the general webx design of using the builder pattern in constructing sql statements instead of strings. 
```java
class Person {
   private String name, email; 
   ....
}

try (
  var con = BlazingDB.create("sqlite", "users.db"); ) {
  List<Person> result = con
    .select(Query.All)
    .from(Query.List("users p ", "emails e"))
    .where(Query.Cond("=", "e.uid", "p.uid"))
    .execute()
    .map(Person.class); 

   .....
} catch (Exception e) { // Do better here
 ....
} 

```

**Comment by hexaredecimal** [comment]
Still have to figure out if this can be affected by sql injections.

**Comment by hexaredecimal** [comment]
ORM Implemented in this commit 992da1bb5a7526479505ec095eb7c9b219b5de50
